src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -23,243 +23,475 @@ internal class BuiltinFunctionsCore
             "RecordInfo", "Relate", "RemoveAll", "RemoveIf", "RequestHide", "Reset", "ResetForm", "Revert", "SaveData", "ScanBarcode", "Select", "SetFocus",
             "SetProperty", "ShowColumns", "State", "SubmitForm", "TraceValue", "Ungroup", "Unrelate", "Update", "UpdateContext", "UpdateIf", "User", "Validate", "ValidateRecord", "ViewForm",
             "Collect", "Clear", "Patch", "Remove", "ClearCollect", "Set"
+        };
+
+        internal static readonly TexlFunction[] __library = new TexlFunction[]

```

Comment:
>__library [](http://example.com/codeflow?start=48&length=9)

nit, single underscore prefix (not double)

Reply:
this was on purpose to avoid a name conflict
I'll rename to avoid the double underscore if that's a concern